bramkouwen/Kouwen_A_Rabari_A_AutomotiveMusicMixer
# adding borders to containers

## What does the PR do?
* It creates the borders to the instrument container and drop area container


## How do you check this PR?
* Pull down the branch
*  See added borders 

## Screenshots 
None.
